### ISSUE TITLE
Action and feedback stop working after companion 2.2.0

### ISSUE THREAD

**Issue by acroweb** [bitfocus/companion-module-skaarhoj-rawpanel#5]
Hello,
Same issue as #2 
I  do the update to go from companion 2.2.0 to 3.2.0 but I have an issue. Only The satellite mode works. When I try to set color of buttons or text on the LCD via action, it doesn't work. Same for feedbacks (there is a comment that says "Currently disabled in the module"). What I have to do to make it work? 

Thank you

**Comment by McHauge** [comment]
Yes this is a limitation of the new internal API's in companion, and the lack of dev time to implement on the 3.x version of the module, the 2.x version "hacked" into alot of internal feedback stuff in order to do alot of the static stuff, and that's just not posible anymore with companion 3.x.

it should still read color and screen text based on how the button looks on a stream deck though, so red will be red and blue will be blue and that stuff. so you will need to build it as if it was a streamdeck for now, or stay on the older version sorry

**Comment by saltmild** [comment]
Hello,
Just discovered this modual and its great, thanks! We have some rackfly units pre-bluepill which we had rather abandoned, which now have a new life with companion.
Just wanted to give this post a little bump so see if a new working system for feedback (led interncity and 4 way button specifically) for V3 has been found? 
Thank you

**Comment by McHauge** [comment]
@saltmild Hi, sorry so far I have not spend alot of time one it since the last message, but I still don't see how I could do it yet.
With out breaking out of the "module" api. however with the way that the satelite API now looks to support more than 32 HWC's we could maybe expand it to allow for more buttons mapped pr setup. But I'll not promise anything, for now.

**Comment by saltmild** [comment]
Hi @McHauge just wondered if there has been any breakthrough on this yet? Many thanks

**Comment by McHauge** [comment]
@saltmild Hi, sadly no, modules are now isolated from the main system, so I can't reallly added that back. sorry

**Comment by McHauge** [comment]
Hi, there is now the option to run Companion Satellite natively on your SKAARHOJ panel, if it's a BluePill inside controller or BluePill server. 

This supports up to 1.000 buttons/encoders pr satellite added on the SKAARHOJ side, I doubt you will hit that. 
For more info pls check this:

https://devices.skaarhoj.com/core/core-bitfocus-companion-satellite?version=pre
https://cloud.skaarhoj.com/coremanual/core-bitfocus-companion-satellite/pre

**Comment by saltmild** [comment]
Ah thats great! Shame its BluePill only, we have a lot of pre bluepill panels and don't always want to bring a bluepill server to site, but that's still very very helpful, thanks

**Comment by McHauge** [comment]
Well, Unisketch is end of life, so to say, it's there as is, with only bug fixes, no active development. So like for those you can still use the companion module. and add more than one of them pr panel if you need more than 32buttons
